### PR TITLE
Css brochure form

### PR DESCRIPTION
**Pull request by DreSean-Sutton**
image

### CODE REVIEW

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
+* {
+  box-sizing: border-box;
+  font-family: BrandonGrotesque-Regular, sans-serif;
+}
+
+body {
+  margin-left: 27rem;
+  margin-right: 27rem;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:8]: Instead of doing this, I suggest setting a `width` on the `form` element, then use `margin: auto` to center it 🤓

**Comment by DreSean-Sutton** [comment]
image

File: css-brochure-form/styles.css [hunk 2]
```diff
@@ -3,10 +3,12 @@
   font-family: BrandonGrotesque-Regular, sans-serif;
 }
 
-body {
-  margin-left: 27rem;
-  margin-right: 27rem;
+html {
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:9]: You should _**never**_ set a width on `html` or `body`

Comment (codingcodymiller) [reply css-brochure-form/styles.css:9]: Set a `width` on the `form` element, like I mentioned 🤓

**Review by codingcodymiller — APPROVED** [review]
Great work!!

**Pull request merged**